lib/MapboxInspect.js

```diff
@@ -213,7 +214,7 @@ MapboxInspect.prototype._onMousemove = function (e) {
 
       var renderedPopup = this.options.renderPopup(features);
 
-      if (typeof renderPopup === 'string') {
+      if (typeof renderedPopup === 'string') {
```

Comment:
This was causing the examples to fail because `renderPopup` is always a function, whereas `renderedPopup` was the intended check.